doc/about/user-guidelines.md

```diff
@@ -0,0 +1,67 @@
+# `mybinder.org` Usage Guidelines
+
+This page details some guidelines and policies that we follow at ``mybinder.org``.
+
+:::{admonition,warning} When in doubt, open an issue
+If you're not sure whether your use-case is reasonable for `mybinder.org`, don't
+hesitate to [open an issue](https://github.com/jupyterhub/mybinder.org-deploy/issues/new?labels=impact&template=request_resources.md)
```

Comment:
```suggestion
hesitate to [open an issue](https://github.com/jupyterhub/mybinder.org-deploy/issues/new?labels=question&template=ban_check.md)
```

Maybe the issue template to check if a repo meets the requirements is more appropriate here?

Reply:
I think this is a better template, but we should call it something other than `ban-check.md`. that will sound hostile to people that are just checking in on whether they have an OK use-case.

Reply:
see https://github.com/jupyterhub/mybinder.org-deploy/pull/1651 @sgibson91 WDYT?

Reply:
Agreed. I'd clearly thought a bit about it since I gave it the title "repo check" but hadn't worked that idea all the way through 😂